interactions/ext/prefixed_commands/manager.py

```diff
@@ -229,17 +228,13 @@ def remove_command(self, name: str, delete_parent_if_empty: bool = False) -> Opt
 
         return command
 
-    @listen("callback_added")
-    async def _register_command(self, event: CallbackAdded) -> None:
+    def _register_command(self, callback: Callable) -> None:
         """Registers a prefixed command, if there is one given."""
-        if not isinstance(event.callback, PrefixedCommand):
+        if not isinstance(callback, PrefixedCommand):
             return
 
-        cmd = event.callback
-        cmd.extension = event.extension
```

Comment:
Note to any reviewer: this line was redundant, since extensions do this themselves before even adding the command.